content/blogs/newsletter-103.md

```diff
@@ -0,0 +1,121 @@
+
+## UEFI Boot
+Thanks to the work of Timo Kreuzer (tkreuzer) our AMD64 is now capable of booting on an increasing number of hardware.
+With this new step we now are able to look at booting on more modern devices, the first step of this transition being the support of UEFI Class 3 boot (Systems without a BIOS fallback).
+Work has been going on since the beginning of the year to transition freeloader our default bootloader for ReactOS to support UEFI on ARM32, ARM64, x86 and of course primarily; AMD64.
+Developer Hermès BÉLUSCA - MAÏTO (HBelusca) has been developing a system for passing the UEFI framebuffer information in a fashion that could be supported even under older systems bringing even XP to UEFI!
+While developer Justin Miller (The_DarkFire_) has been developing the UEFI freeloader build.
+
+On top of just supporting booting ReactOS some other features are being built such as a bootmgfw compatible disguised build of FreeLdr so modern Windows 11 systems can bootstrap FreeLdr easily as well as EFI chainloading giving boot management capabilities to our favorite bootloader.
+
+{{< gallery >}}
+{{< figure link="/img/blogs/newsletter-103/uefi.png" src="/img/blogs/newsletter-103/uefi.png" caption="Running ReactOS on a Steam Deck" >}}
+{{< /gallery >}}
+
+**WARNING:** UEFI support in ReactOS is experimental, under heavy development and not yet ready for the general use. We currently cannot give an estimation as of when it will be done, so please stay tuned!
+
+## Shell improvements
+Doug Lyons, Katayama Hirofumi MZ and Carl Bialorucki (cbialorucki) are extensively improving the Shell component of ReactOS to provide a better user experience and sleek interaction with the operating system.
+Notably Doug Lyons improved the toolbar icon display in MS Office 2000 and other programs, by fixing some bugs in the internal functions (`SetDIBitsToDevice` and `NtGdiSetDIBitsToDeviceInternal`).
+
+{{< gallery >}}
+{{< figure link="/img/blogs/newsletter-103/before.png" src="/img/blogs/newsletter-103/before.png" caption="BEFORE" >}}
+{{< figure link="/img/blogs/newsletter-103/after.png" src="/img/blogs/newsletter-103/after.png" caption="AFTER" >}}
+{{< /gallery >}}
+
+Katayama improved the ZIP functionality support to allow the possibility to zip or unzip files or folders that have UTF-8 characters in them.
+This will greatly help with the interoperability with other systems of different language settings.
+In addition to that, we can choose the code-page of ZIP compression/extraction by using the following registry key (`HKCU\Software\ReactOS:ZipCodePage / UnZipCodePage`).
+
+{{< gallery >}}
+{{< figure link="/img/blogs/newsletter-103/utf8.png" src="/img/blogs/newsletter-103/utf8.png" caption="Extracting from a ZIP file that has Japanese characters on a Chinese locale system">}}
+{{< /gallery >}}
+
+Katayama added a "Internet Browser" icon in the desktop workspace, which it will open the current and default web browser that is installed onto the computer.
```

Comment:
```suggestion
Katayama added an "Internet Browser" icon in the desktop workspace, which it will open the current and default web browser that is installed onto the computer.
```